pages/edge/tutorials/cdn-on-wasmer-edge.mdx

```diff
@@ -0,0 +1,357 @@
+# kind = 302
+
+### URL Rewrites (examples only)
+
+# [[advanced.rewrites]]
+# source = "**/*.{png,ico,gif}"
+# destination = "/assets/favicon.ico"
+## Optional redirection
+# redirect = 301
+
+# [[advanced.rewrites]]
+# source = "**/*.{jpg,jpeg}"
+# destination = "/images/sws.png"
+
+### Virtual Hosting
+
+# [[advanced.virtual-hosts]]
+## But if the "Host" header matches this...
+# host = "sales.example.com"
+## ...then files will be served from here instead
+# root = "/var/sales/html"
+
+# [[advanced.virtual-hosts]]
+# host = "blog.example.com"
+# root = "/var/blog/html"
+```
+
+<Callout type="info" emoji="ℹ️">
+  The above configuration is the configuration of the `static-web-server`
+  package. More info about it can be found
+  [here](https://static-web-server.net/configuration/config-file/).
+</Callout>
+
+Now, let's add the following configuration to the `wasmer.toml` file.
+
+```yaml
+[package]
+name = "<your-namespace>/<your-cdn-name>"
+version = "0.1.0"
+description = "<your-namespace>/<your-cdn-name> CDN on Wasmer Edge"
+
+[dependencies]
+"wasmer/static-web-server" = "^1"
+
+[fs]
+public = "public"
+config = "config" # 👈🏼 Add the config directory
+
+[[command]]
+name = "script"
+module = "wasmer/static-web-server-async:webserver"
+runner = "https://webc.org/runner/wasi"
+
+[command.annotations.wasi] # 👈🏼 Add config.toml as an argument for the webserver
+main-args = ["-w", "./config/config.toml"]
+```
+
+### Adding files to the CDN
+
+Now, let's add some files to the `public` directory.
+
+```bash
+$ touch public/index.html
+```
+
+I'll also add an image called `image.jpg` to the `public` directory.
+
+And now my `index.html` file looks like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <img src="./image.jpg" width="600" />
+  </body>
+</html>
+```
+
+You can also add other files like css, js, videos, pdfs, etc.
+
+You can view my app [here](https://dynamite-bud-sws-cdn.wasmer.app).
+
+### Deploying the CDN
+
+Now, let's deploy the CDN.
+
+```bash
+$ wasmer deploy
+```
+
+The above command will prompt you with the following:
+
+```bash
+Loaded app from: /path/to/your/dir/app.yaml
+
+Publish new version of package '<your-namespace>/<your-cdn-name>'? yes
+Publishing package...
+[1/2] ⬆️   Uploading...
+[2/2] 📦  Publishing...
+Successfully published package `<your-namespace>/<your-cdn-name>@0.1.0`
+Waiting for package to become available.....
+Package '<your-namespace>/<your-cdn-name>@0.1.0' published successfully!
+
+Deploying app <your-namespace>-<your-cdn-name>...
+
+ ✅ App <your-namespace>/<your-namespace>-<your-cdn-name> was successfully deployed!
+
+> App URL: https://dynamite-bud-sws-cdn.wasmer.app
+> Versioned URL: https://2qjh9iec55z4.id.wasmer.app
+> Admin dashboard: https://wasmer.io/apps/dynamite-bud-sws-cdn
```

Comment:
This url should not have anything about your user here. It should also be visitable by people